src/main/java/at/hannibal2/skyhanni/config/features/mining/MiningConfig.java

```diff
@@ -13,6 +13,10 @@ public class MiningConfig {
     @Category(name = "Mining Event Tracker", desc = "Settings for the Mining Event Tracker")
     public MiningEventConfig miningEvent = new MiningEventConfig();
 
+    @Expose
+    @Category(name = "Hotm", desc = "Settings for Heart of the Mountain")
```

Comment:
```suggestion
    @Category(name = "HotM", desc = "Settings for Heart of the Mountain")
```
I think all instances where the users sees the word "Hotm" should be replaced with "HotM" since it in theory makes more sense
It doesnt really need changing in the code tho